requirements.txt

```diff
@@ -1,4 +1,6 @@
 -e git+http://github.com/xamox/pygame
 MySQL-python==1.2.5
-Django==1.6.6
+Django==1.7
```

Comment:
I'll give it a shot after going through the rest of the edits. I'll add it to our Trello board.